apps/test/unit/p5lab/spritelab/TextConsoleTest.js

```diff
@@ -4,6 +4,8 @@ import TextConsole from '@cdo/apps/p5lab/spritelab/TextConsole';
 import {mount} from 'enzyme';
 import sinon from 'sinon';
 
+const AUTO_CLOSE_TIME = 4000;
```

Comment:
nit: i usually export constants like this from the component being tested so that it only needs to be changed in one place